eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgAuditDb.scala

```diff
@@ -38,18 +40,28 @@ class PgAuditDb(implicit ds: DataSource) extends AuditDb with Logging {
   import ExtendedResultSet._
 
   val DB_NAME = "audit"
-  val CURRENT_VERSION = 4
+  val CURRENT_VERSION = 5
 
   case class RelayedPart(channelId: ByteVector32, amount: MilliSatoshi, direction: String, relayType: String, timestamp: Long)
 
   inTransaction { pg =>
     using(pg.createStatement()) { statement =>
+      def migration45(statement: Statement): Int = {
+        statement.executeUpdate("CREATE TABLE IF NOT EXISTS relayed_trampoline (payment_hash TEXT NOT NULL, amount_msat BIGINT NOT NULL, next_node_id TEXT NOT NULL, timestamp BIGINT NOT NULL)")
```

Comment:
@pm47 said something recently about not using `BIGINT` for timestamps in postgres, not sure if that's something you should change in this PR or whether he'll do a sweep of all the codebase to update it.

Reply:
Indeed, but it'll take some time so let's use `BIGINT` here and we'll migrate all of them in one pass.

Reply:
It's unrelated and everything else still uses `BIGINT`. We should change all the timestamps together if we decide we want to do it.